### DISCUSSION TITLE
IE11兼容性问题

### DISCUSSION THREAD

**Discussion by gaolongge** [antvis/G2#5373] [Q&A] [answered]
问题：
 IE11不兼容，导致页面报错，无法加载；babel配置：https://cli.vuejs.org/zh/guide/browser-compatibility.html；
且文档也没有兼容性的描述；

版本：
  "@antv/g2": "^4.2.8"

报错如下：
   
![WechatIMG300](https://github.com/antvis/G2/assets/24975204/9ed5d7ad-ad50-488d-bc74-8cb698cd9c61)
![WechatIMG299](https://github.com/antvis/G2/assets/24975204/988d4a8d-983a-4afe-ad39-a1c257150307)

**Comment by hustcc** [comment]
代码在这里：https://github.com/antvis/d3-color/blob/1a74dbc962d042ef6456b4b2e2f6a97847d55b15/src/color.js#L15

是 d3-color 在 ie11 上存在有些问题！

**Comment by xiaoiver** [comment, marked as answer]
d3-color 源码使用 \` 并没有问题，它提供的构建产物也都进行了转译：

<img width="1257" alt="截屏2023-08-01 上午11 19 24" src="https://github.com/antvis/G2/assets/3608471/56490b65-11a9-481d-a5ce-8181d97a0b1a">

方便提供一些更详细的项目信息吗？比如使用的是 Vite 还是 Webpack？

**Reply by gaolongge** [reply]
使用的是webpack。是G2内部引用了d3-color，导致的。目前已经解决，babel转化一下就行了
